<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(37, 41, 50);
color: rgb(255, 255, 255);
font: 16pt &quot;Karla&quot;;</string>
  </property>
  <widget class="QPushButton" name="btnSalir">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(37, 41, 50);
font: 12pt &quot;Karla&quot;;</string>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserta Producción Gramatical</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="noTerminalSimbol">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Karla</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Karla&quot;;
color: rgb(255, 255, 255);
 border-radius: 15px;
background-color: rgb(25, 28, 35);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="productionInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Karla&quot;;
color: rgb(255, 255, 255);
 border-radius: 15px;
background-color: rgb(25, 28, 35);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(37, 41, 50);
font: 12pt &quot;Karla&quot;;</string>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelProducciones">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
background-color: rgb(46, 52, 63);
color: gray;
text-align: center;
padding: 6px;
justify-content: center;
align-items: center;</string>
   </property>
   <property name="text">
    <string>V → Producción</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalir_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(37, 41, 50);
font: 12pt &quot;Karla&quot;;</string>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Karla&quot;;</string>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Karla&quot;;</string>
   </property>
   <property name="text">
    <string>Producción</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
